code/search/SearchVariantSubsites.php

```diff
@@ -83,9 +83,16 @@ public function extractManipulationWriteState(&$writes)
                 continue;
             }
 
+            if (isset($write['fields']['SiteTree:SubsiteID'])) {
```

Comment:
Yes but files in subsiteid = 3 shouldn't be.

If they are global, they must be indexed in EVERY subsite, not only once.